### ISSUE TITLE
Build failure of development version

### ISSUE THREAD

**Issue by subhacom** [NeuroML/jNeuroML#3]
When trying to install the development version as described in README.md, the build exits with error:

```
...

>>>  Executing: (mvn install) in dir: ../org.neuroml.export
Traceback (most recent call last):
  File "getNeuroML.py", line 83, in <module>
    info = execute_command_in_dir(command, local_dir)
  File "getNeuroML.py", line 50, in execute_command_in_dir
    return_string = subprocess.check_output("cd %s %s%s"%(directory, sep, command), shell=True)
  File "/usr/lib/python2.7/subprocess.py", line 544, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
subprocess.CalledProcessError: Command 'cd ../org.neuroml.export  ; mvn install' returned non-zero exit status 1
```

When I run the command manually, I get this:

````
user@localhost$ cd ../org.neuroml.export  ; mvn install
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building org.neuroml.export 1.0.9
[INFO] ------------------------------------------------------------------------
[WARNING] The POM for org.neuroml.model:org.neuroml.model:jar:1.0.4 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.354s
[INFO] Finished at: Sat May 11 15:44:46 IST 2013
[INFO] Final Memory: 3M/118M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project org.neuroml.export: Could not resolve dependencies for project org.neuroml.export:org.neuroml.export:jar:1.0.9: Failure to find org.neuroml.model:org.neuroml.model:jar:1.0.4 in http://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
````

On this system "java -version" shows:

Using java version "1.6.0_27" 
OpenJDK Runtime Environment (IcedTea6 1.12.3) (6b27-1.12.3-0ubuntu1~12.04.1)
OpenJDK 64-Bit Server VM (build 20.0-b12, mixed mode)

**Comment by theiera** [comment]
I have encountered the same issue raised in the beginning by @subhacom.
I had manually set the current branch in jNeuroML to "development" while I was running the command to build the master branch:
python getNeuroML.py
Setting, again by hand, the current branch in jNeuroML to master the compilation would successfully complete.

I did also a crosscheck: jNeuroML set to master and build using
python getNeuroML.py development
fails to build.

Finally, setting the jNeuroML to development branch and running:
python getNeuroML.py development
successfully completed.

Is this a bug of a feature?
